srcs/app/server/WebServer.cpp

```diff
@@ -9,14 +11,9 @@ WebServer::WebServer() {
 	max_sd_ = -1;
 }
 
-bool	WebServer::Init(const std::string &pathname) {
-	// Load the config file
-	if (!config_.LoadFile(pathname))
-		return false;
-	// Populate the servers
-	if (!PopulateServers_())
-		return false;
-	return true;
+void	WebServer::Init(const std::string &pathname) {
```

Comment:
I assume, with this new approach of dispatching errors as exceptions, you already manage `PopulateServers` errors withing callable methods but, what happens with `LoadFile`? I assume dispatching exceptions will be responsability of the `Config` or the `ServerConfig` class maybe?

Reply:
Great question, and yes, the dispatching of errors should be the responsibility of the `Config` class, either by not catching the exceptions or catching and rethrowing (if it needs to clean up some resources)